misc/docs/source/tutorial/code_snippets/quakeml_custom_tags.rst

```diff
@@ -4,12 +4,12 @@
 Handling custom defined tags in QuakeML and the ObsPy Catalog/Event framework
 =============================================================================
 
-QuakeML allows use of custom elements in addition to the "usual" information
+QuakeML allows use of custom elements in addition to the 'usual' information
 defined by the QuakeML standard. It allows *a)* custom namespace attributes to
 QuakeML namespace tags and *b)* custom namespace subtags to QuakeML namespace
 elements.
-ObsPy can handle both basic (non-nested) custom tags in event type objects
-(*a)*) and custom attributes (*b)*) during input/output to/from QuakeML.
+ObsPy can handle both basic custom tags in event type objects (*a)*) and custom
```

Comment:
Oh, I get it; it's a back-reference. In that case, remove the interior parenthesis instead.